config/config.go

```diff
@@ -426,6 +434,35 @@ type Checkpoint struct {
 	Hash   types.Hash `toml:"hash" comment:"checkpoint block hash."`
 }
 
+type Erc20RewardSignerConfig struct {
+	Enable      bool           `toml:"enable" comment:"enable the ERC20 reward signer service"`
+	Targets     []string       `json:"targets" comment:"target reward ext alias for the ERC20 reward"`
+	PrivateKeys []string       `json:"private_keys" comment:"private key for the ERC20 reward target"`
```

Comment:
here I made this an array, so different 'signer' could be configured for different target